qemu/tests/cfg/boot_nic_with_iommu.cfg

```diff
@@ -0,0 +1,17 @@
+- boot_nic_with_iommu:
+    x86_64, i386:
+        only q35
+        no WinXP WinVista Win7 Win8 Win8.1 Win2003
+        no Win2008 Win2008..r2 Win2012 Win2012..r2
+        machine_type_extra_params = "kernel-irqchip=split"
+        HostCpuVendor.intel:
+            intel_iommu = yes
+        Linux:
+            enable_guest_iommu = yes
```

Comment:
What will it happen for amd ? No intel-iommu device, but enable guest iommu ? Is this the expected ?

Reply:
Left this comment. On AMD, only enable iommu_platform for all virtio devices. `enable_guest_iommu` will check the CPU vendor at first and then add a parameter to the kernel command line.

Reply:
Better  'enable_guest_iommu = yes'  with 'HostCpuVendor.intel'  only,   But anyway, patch already be merged and result is same, leave it ~